addons/luamodel/CMakeLists.txt

```diff
@@ -6,7 +6,7 @@ SET_TARGET_PROPERTIES ( ${PROJECT_EXECUTABLES} PROPERTIES
   LINKER_LANGUAGE CXX
   )
 
-FIND_PACKAGE (Lua51 REQUIRED)
+FIND_PACKAGE (Lua 5.1 REQUIRED)
```

Comment:
It seems that you've based these changes on your PR of the lua fix. 
It would be better that this PR is standalone.
Let's see that we can merge the other PR first, such that you can rebase this PR first.